motoren = [
    {
        "name"    : "036LO44U",
        "M_cur_P_gain"  : 1,
        "M_cur_I_gain"  : 1,
        "M_vel_P_gain"  : 3,
        "M_vel_I_gain"   : 60,
        "M_pos_P_gain"  : 1,
        "M_pos_I_gain"  : 60,
        "M_pos_D_gain"  : 0.1,
        "M_pos_saturation"  : 1,
        "M_L"  : 21.0,
        "M_R"  : 1.05,
        "M_peak_I"  : 3850,
        "M_rated_I"  : 2150,
        "M_max_I"  : 1000,
        "M_rated_torque"  : 1450,
        "M_torque_slope"  : 10000,
        "M_max_speed"  : 6000,
        "M_max_profile_speed"  : 6000,
        "M_max_acc"  : 5000,
        "M_max_dec"  : 5000,
        "M_max_profile_acc"  : 3000,
        "M_max_profile_dec"  : 3000
    },
    {
        "name"    : "045LO64U",
        "M_cur_P_gain"  : 1,
        "M_cur_I_gain"  : 1,
        "M_vel_P_gain"  : 3,
        "M_vel_I_gain"   : 60,
        "M_pos_P_gain"  : 1,
        "M_pos_I_gain"  : 60,
        "M_pos_D_gain"  : 0.1,
        "M_pos_saturation"  : 1,
        "M_L"  : 3.5,
        "M_R"  : 0.91,
        "M_peak_I"  : 6900,  # Motor Peak Current [mA] , UINT32
        "M_rated_I"  : 4700,  # Motor Rated Current [mA], UINT32
        "M_max_I"  : 1000,
        "M_rated_torque"  : 3800,  # Motor Rated Torque [mNm], UINT32
        "M_torque_slope"  : 10000,
        "M_max_speed"  : 6000,
        "M_max_profile_speed"  : 6000,
        "M_max_acc"  : 5000,
        "M_max_dec"  : 5000,
        "M_max_profile_acc"  : 3000,
        "M_max_profile_dec"  : 3000
    },    
    {
        "name"    : "045MO64U",
        "M_cur_P_gain"  : 1,
        "M_cur_I_gain"  : 1,
        "M_vel_P_gain"  : 3,
        "M_vel_I_gain"   : 60,
        "M_pos_P_gain"  : 1,
        "M_pos_I_gain"  : 60,
        "M_pos_D_gain"  : 0.1,
        "M_pos_saturation"  : 1,
        "M_L"  : 4.7,
        "M_R"  : 1.4,
        "M_peak_I"  : 5200,
        "M_rated_I"  : 3600,
        "M_max_I"  : 1000,
        "M_rated_torque"  : 2900,
        "M_torque_slope"  : 10000,
        "M_max_speed"  : 6000,
        "M_max_profile_speed"  : 6000,
        "M_max_acc"  : 5000,
        "M_max_dec"  : 5000,
        "M_max_profile_acc"  : 3000,
        "M_max_profile_dec"  : 3000
    },    
    {
        "name"    : "056LO64U",
        "M_cur_P_gain"  : 1,          # Current P-Gain [0-100]
        "M_cur_I_gain"  : 1,          # Current I-Gain [0-100]
        "M_vel_P_gain"  : 3,          # Velocity Proportional (KP) 
        "M_vel_I_gain"   : 60,         # Velocity I-Gain [Hz]
        "M_pos_P_gain"  : 1,          # Position Controller P-Gain [0-1200]
        "M_pos_I_gain"  : 60,         # Position Controller I-Gain [0-1000]
        "M_pos_D_gain"  : 0.1,          # Position Controller D-Gain [0-1000]
        "M_pos_saturation"  : 1,          # Position Integral Output Saturation [pos]
        "M_L"  : 2.0,        # Motor Inductance [MHz] , REAL32 *2 ? 
        "M_R"  : 0.48,       # Motor Resistance [Ohm] , REAL32 *2 ? 
        "M_peak_I"  : 17000,      # Motor Peak Current [mA] , UINT32
        "M_rated_I"  : 5600,       # Motor Rated Current [mA], UINT32
        "M_max_I"  : 1000,       # Maximum Current [MICONT], Rated Current / 1000
                                # =>  Rated Current / 1000 * [0x6073]; also bei 1000 ist MAX_CURR = Motor Rated Current
        "M_rated_torque"  : 14000,      # Motor Rated Torque [mNm], UINT32
        "M_torque_slope"  : 10000,      # Torque Slope [mNm/s], UINT32
        "M_max_speed"  : 6000,       # Max Motor Speed [rpm], UINT32 
        "M_max_profile_speed"  : 6000,       # Max Profile Velocity [user]
        "M_max_acc"  : 5000,       # Max Acceleration [user]  
        "M_max_dec"  : 5000,       # Max Deceleration [user]
        "M_max_profile_acc"  : 3000,       # Profile Acceleration [user]
        "M_max_profile_dec"  : 3000        # Profile Deceleration [user]
    }
]
       